recipes/msmpi/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version="7.0.12437.6" %}
+
+package:
+  name: msmpi
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True  # [not win]
+
+requirements:
+  run:
+    - msmpi-bin {{ version }}
+    - msmpi-sdk {{ version }}
+
+test:
+  commands:
+    # tests are in bin and sdk packages, not here
+    - echo
+
+about:
+  home: https://msdn.microsoft.com/en-us/library/windows/desktop/bb524831(v=vs.85).aspx
+  summary: metapackage for Microsoft MPI programs and libraries
+  license: proprietary
```

Comment:
Would be nice to have a license file here too. Also echoing @ocefpaf's question about whether we can distribute this.